pkgs/servers/go-autoconfig/add_gosum.patch

```diff
@@ -0,0 +1,76 @@
+diff --git a/go.mod b/go.mod
+index 72f479d..4fc2c09 100644
+--- a/go.mod
++++ b/go.mod
+@@ -1,13 +1,23 @@
+ module go-autoconfig
+ 
++go 1.17
```

Comment:
go 1.17 is EOL

Reply:
I updated the package and it is now using go 1.18. If this is still to old or EOL I could request a update upstream